e2e/decomission/decomission_test.go

```diff
@@ -51,10 +49,13 @@ func TestMain(m *testing.M) {
 	os.Exit(code)
 }
 
+// TestDecomissionFunctionality creates a cluster of 4 nodes and then decomissions on of the CRDB nodes.
+// It then checks that the cluster is stable and that decomissioning is successful.
 func TestDecommissionFunctionality(t *testing.T) {
 
 	// Testing removing and decommisioning a node.  We start at 4 node and then
 	// remove the 4th node
+	utilfeature.DefaultMutableFeatureGate.Set("UseDecommission=true")
```

Comment:
@udnay we turn on Decommission here.

Reply:
Ah, I missed it. Thought it was unit-tests.

Reply:
Actually do any of the other `e2e` tests assume decommission is on?

Reply:
@udnay no they do not.  We have had the decommission actor cause problems in other e2e tests, so I could enable it everywhere.  What do you think?